src/Shared/Recruit.Vacancies.Client/Application/Aspects/RequestPerformanceBehaviour.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.Logging;
+
+namespace Esfa.Recruit.Vacancies.Client.Application.Aspects
+{
+    public class RequestPerformanceBehaviour<TRequest, TResponse> : IPipelineBehavior<TRequest, TResponse>
+    {
+        private readonly Stopwatch _timer;
+        private readonly ILogger<TRequest> _logger;
+
+        public RequestPerformanceBehaviour(ILogger<TRequest> logger)
+        {
+            _timer = new Stopwatch();
+
+            _logger = logger;
+        }
+
+        public async Task<TResponse> Handle(TRequest request, CancellationToken cancellationToken, RequestHandlerDelegate<TResponse> next)
+        {
+            _timer.Start();
```

Comment:
can you not just use `var timer = Stopwatch.StartNew();` in this method instead of having a class level field?

Reply:
🥇